src/mode/logtalk_highlight_rules.js

```diff
@@ -170,7 +158,8 @@ var LogtalkHighlightRules = function() {
          { token: 'punctuation.definition.string.begin.logtalk',
            regex: '"',
            push: 
-            [ { token: 'constant.character.escape.logtalk', regex: '\\\\.' },
+            [ { token: 'constant.character.escape.logtalk',
+                regex: '\\\\([\\\\abfnrtv"\']|(x[a-fA-F0-9]+|[0-7]+)\\\\)' },
```

Comment:
should we consider other escape sequences like `\UXXXXXXXX` or `\uXXXX`?

Reply:
Those escape sequences are implemented only by some of the backend Prolog compilers supported by Logtalk, notably only for those that support Unicode. They are slowly becoming a de facto standard feature. But they are also rarely used.